docs/content/sre/machinepools/machinepools.md

```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
+
+| vCPU | Memory |
+|---|---|
+| 4  | 32 GiB  |
+
+## 1x Infra
+
+The following infrastructure workloads do not incur SAAP worker subscriptions, at least one infrastructure node is required for deployments.
+
+|  | vCPU | Memory |
+|---|---|---|
+| Ingress monitor controller  | 150m  | 600 MiB  |
+| OpenShift-GitOps  | 530 m  | 500 MiB  |
+| Stakater-nexus  | 200 m  | 1.6 GiB  |
+| Vault  | 255 m  | 360 MiB  |
+|  Stakater-Tronador  | 100 m  | 200 MiB  |
+|  OpenShift-Velero  | 500 m  | 150 MiB  |
+|  Stakater-tenant-operator  | 600 m  | 1.2 GiB  |
+|  Stakater-Forecastle  | 50 m  | 200 MiB  |
+|  Stakater-SonarQube  | 350 m  | 1.5 GiB  |
+
+| vCPU | Memory |
+|---|---|
+| 4  | 16 GiB  |
+## 1x Logging
 
-## 3x Infra
+For Cluster aggregated logging , we offer one dedicated logging machine. Minimum requirements for logging infrastructure is as follows: 
 
-We use three infrastructure machines for all the resources except for logging , monitoring or pipelines workloads.
+|  | vCPU | Memory |
+|---|---|---|
+| OpenShift logging | 4  | 12 GiB  |
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  | 8  | 32 GiB  | 300 USD  |
-|  Binero | 8 | 32 GB  | 2079.88 SEK  |
+| vCPU | Memory |
+|---|---|
+| 4  | 16 GiB  |
 
-## 1x Logging
+## 1x Monitoring
 
-One logging machine is used for cluster logging workloads.
+SAAP exposes metrics that can be collected and stored in back-ends by the cluster-monitoring-operator. As an SAAP administrator, you can view system resources, containers and components metrics in one dashboard interface, Grafana. Exactly one monitoring nodes must be used for all production deployments. The recommended size of a monitoring node is the minimum of 6 vCPUs and 14 GB ram per 7200 pods.
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  | 4  | 16 GiB  | 150 USD  |
-|  Binero | 4  | 16 GB | 880.04 SEK  |
+|  | vCPU | Memory |
+|---|---|---|
+| OpenShift monitoring | 600 m  | 3 GiB  |
+| Stakater-workload-monitoring | 1 GiB  | 4 GiB  |
 
-## 2x Monitoring
+| vCPU | Memory |
+|---|---|
+| 6 | 24 GiB  |
 
-Two monitoring machines are used for cluster monitoring workloads
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  |8  | 32 GiB  | 300 USD  |
-|  Binero | 6  | 32 GB  | 1640.09 SEK  |
+## 1x Pipeline
 
+Red Hat OpenShift Pipelines is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. You can use the SAAP Developer console to create Tekton resources, view logs of pipeline runs, and manage pipelines in your SAAP namespaces. We offer one dedicated pipeline machine. Minimum requirements for pipeline infrastructure is as follows: 
```

Comment:
Once again, we offer or we require?

Also link to other doc sources if there exists